test/src/main/java/org/apache/accumulo/test/WriteAfterCloseIT.java

```diff
@@ -141,12 +142,24 @@ public void testWriteAfterClose(TimeType timeType, boolean killTservers, long ti
     try (AccumuloClient c = Accumulo.newClient().from(props).build()) {
       c.tableOperations().create(table, ntc);
 
-      List<Future<?>> futures = new ArrayList<>();
-
-      for (int i = 0; i < 100; i++) {
-        futures.add(
-            executor.submit(createWriteTask(i * 1000, c, table, timeout, useConditionalWriter)));
+      int numTasks = 100;
+      List<Future<?>> futures = new ArrayList<>(numTasks);
+      CountDownLatch startLatch = new CountDownLatch(32); // synchronize start of a portion of the
+                                                          // tasks
```

Comment:
```suggestion
      // synchronize start of a portion of the tasks
      CountDownLatch startLatch = new CountDownLatch(32);
```

Reply:
why not all of them?

Reply:
I've found that waiting for many many threads in this way causes issues with the test hanging. I think setting an upper limit in these cases makes sense since it still adds the value of starting many at once but doesn't cause the hanging issues.

Reply:
Did you find out why this caused hangs?

Reply:
Not exactly. I think its just contention between a relatively small number of threads trying to wait for a very large number of operations to be at the same spot.